utils/navigator_msgs/msg/Alarm.msg

```diff
@@ -1,4 +1,4 @@
-# Sub8 Alarm message. All alarms should be published on the /alarm topic
+# SubjuGator Alarm message. All alarms should be published on the /alarm topic
 
```

Comment:
Can you leave a comment here saying this is deprecated and alarms should be ported to ROS alarms before further development is done?